woocommerce/woocommerce-ios
# Dynamic Dashboard: Add new cards badge and New Cards Notice card.

Closes: #12860


## Description


This PR adds a way to nudge users that there are new cards. To avoid disrupting user's flow, these cards are hidden by default, and they can be enabled manually inside the Customize screen. The nudges are done via:
1. Adding a purple dot badge next to the "Edit" button,
2. Adding a new, temporary "card" saying "Looking for more insights?", with a button to open Customize screen.

Both the dot and temporary card are removed automatically once a user opens the Customize screen.

(CI says there's a unit test issue but it seems unrelated to this PR)

## Testing instructions


1. Start from a logged out state to clear up saved cards, which will be needed for this test.
4. Log in,
5. In Dashboard, ensure the Edit button on top right has a purple dot badge as a notice. Check screenshot.
6. Scroll down if needed and ensure you see a card that says "Looking for more insights?". Check screenshot.
7. Tap "Edit" button on top right to open the Customize screen.
8. Dismiss the Customize screen, ensure the purple badge and the "Looking for more insights?" card are now gone.
9. Switch to a new site (or log out and log back in again), 
10. Tap the "Add new sections" button in the "Looking for more insights" card, ensure it opens the Customize screen.
11. Dismiss the Customize screen, ensure the card and badge are now gone.

## Screenshots


Check out the badge next to the "Next" button, and the "Looking for more insights" card at the bottom:

<img src="https://github.com/woocommerce/woocommerce-ios/assets/266376/52ab8e12-b891-4d20-b010-a36996c600b6" width="42%">
